### DISCUSSION TITLE
Latest Full Screen Update

### DISCUSSION THREAD

**Discussion by zero-mindset** [daksh2k/Spicetify-stuff#87] [General]
I’m not super familiar with GitHub so I don’t know if this is the best place to put this, but the latest update for Full Screen is great. The ability to slow down the animated background is chef’s kiss. Putting it down to 5% has this extension the closest any fullscreen look on Spotify has ever come to Apple Music’s which is imo the greatest. This extension keeps getting better and better with every update. So, I just wanted to say thanks for helping make Spotify desktop a bearable app to use!

**Comment by daksh2k** [comment]
Glad that you like the extension, always good to hear it !

**Reply by samuelawachie** [reply]
> Glad that you like the extension, always good to hear it !
>
Is there a way we can know or see what exactly these changes do that were added recently? These 3 I mean
1. [feat(full-screen): add queue in full screen](https://github.com/daksh2k/Spicetify-stuff/commit/6749e355ceacb4cd5d5a6fc4f18109214f3989d3)
2. [feat(full-screen): add esc key listener through fullscreenchange](https://github.com/daksh2k/Spicetify-stuff/commit/3dfaeb8ce5f32359ebbb66565c1fdffe37644244)
3. [feat(full-screen): add mousetrap record poc](https://github.com/daksh2k/Spicetify-stuff/commit/f78c04e13fbbcae2db654dd69c314aa9101a7340)
What do they add to the application?
